default.config.yml

```diff
@@ -230,6 +230,11 @@ php_max_input_vars: "4000"
 php_enable_php_fpm: true
 php_fpm_listen: "127.0.0.1:9000"
 
+# If you want to run php-fpm as a user/group other than your OS's default user,
```

Comment:
Instead of adding in this file, can you just add a note on this docs page (http://docs.drupalvm.com/en/latest/other/webservers/#using-nginx) mentioning these two variables and how to set them if switching to `nginx`? Since it's a smaller use case (at least at this point in time), I'd rather not add the four lines to `default.config.yml`.